run/manage.py

```
@@ -0,0 +1,29 @@
+import os
+import sys
+
+# Add the parent directory to the Python path so we can import core
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+is_testing = "test" in sys.argv
+if is_testing:
+    import coverage
+
+    cov = coverage.coverage(source=["app"], omit=["*/tests/*"])
+    cov.set_option("report:show_missing", True)
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)